### PR TITLE
Super punch card

### PR DESCRIPTION
**Pull request by lvdoorn**
Added a rough version of the new punch card.

### CODE REVIEW

File: src/modules/super-punch-card.js [hunk 1]
```diff
@@ -141,7 +141,7 @@ define(function () {
                     return ['Sunday', 'Monday', 'Tuesday', 'Wednesday',
                             'Thursday', 'Friday', 'Saturday'][(d + today + 1) % 7];
                 })
-                .scale(yScale.copy());
+                .scale(d3.scale.linear().domain([6, 0]));
```
Comment (agudek) [inline src/modules/super-punch-card.js]: Is there a reason for changing this line?

Comment (mboom) [inline src/modules/super-punch-card.js]: @agudek Yes, x-axis labels are now visible. Y-axis labels are not aligned but I work on that.

Comment (agudek) [inline src/modules/super-punch-card.js]: The x-axis would remain visible without this change. Change the margin-bottom.

File: src/TimeHelper.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+    this.getDayOfTimestamp = function (timestamp) {
+        for (var i = 0; i < weekday.length; ++i) {
+            var timespan = this.getTimespanOfDay(weekday[i]);
+            if (timespan.start.getTime() <= timestamp.getTime() &&
+                timestamp.getTime() <= timespan.end.getTime()) {
+                return weekday[i];
+            }
+        }
+        return 'Out of scope';
```
Comment (DaanvanderValk) [inline src/TimeHelper.js]: Is this "out of scope" tested for when using this method?

**Pull request merged**